Solutions/Palo Alto - XDR (Cortex)/Playbook/azuredeploy.json

```diff
@@ -0,0 +1,480 @@
+                                        ]
+                                    },
+                                    "type":  "ApiConnection",
+                                    "inputs":  {
+                                        "body":  {
+                                            "messageBody":  "\u003cp\u003eHere's the 12 hours acitivity done on host for incident -@{triggerBody()?['object']?['properties']?['incidentNumber']} and host - @{items('For_each')?['HostName']}\u003cbr\u003e\n\u003cbr\u003e\n\u003cbr\u003e\n@{body('Run_query_and_list_results')?['value']}\u003cbr\u003e\n\u003c/p\u003e",
+                                            "recipient":  {
+                                                "channelId":  "19:Ai0dUOSmL9BSgkBzZ1zXAHi3KR-DKRVHDX1UoaBy0141@thread.tacv2",
+                                                "groupId":  "12249569-98eb-4605-aed7-90a78e7cded3"
+                                            }
+                                        },
+                                        "host":  {
+                                            "connection":  {
+                                                "name":  "@parameters('$connections')['teams']['connectionId']"
+                                            }
+                                        },
+                                        "method":  "post",
+                                        "path":  "/beta/teams/conversation/message/poster/Flow bot/location/@{encodeURIComponent('Channel')}"
+                                    }
+                                },
+                                "Run_query_and_list_results":  {
+                                    "runAfter":  {
+                                    },
+                                    "type":  "ApiConnection",
+                                    "inputs":  {
+                                        "body":  "search '@{items('For_each')?['HostName']}'\n| summarize by $table, FileName, DisplayName, VendorName, AlertSeverity, ActionType, InitiatingProcessFileName, PublicIP, RemediationSteps, ProviderName\n| limit 10",
+                                        "host":  {
+                                            "connection":  {
+                                                "name":  "@parameters('$connections')['azuremonitorlogs']['connectionId']"
+                                            }
+                                        },
+                                        "method":  "post",
+                                        "path":  "/queryData",
+                                        "queries":  {
+                                            "resourcegroups":  "MSSP_Prod",
+                                            "resourcename":  "Sentinel-LAW",
+                                            "resourcetype":  "Log Analytics Workspace",
+                                            "subscriptions":  "75d4ea73-xxxx-xxxx-xxxx-e57cxxxx43a",

```

Comment:
There should be clear instructions in the README to modify these. By default, the values should empty or taken as parameters while deployment.